### PR TITLE
test(kubeflow): add kubeflow-pipelines-manifests test

### PR DESCRIPTION
**Pull request by Bobgy**
Test config for https://github.com/kubeflow/pipelines/pull/5390

### CODE REVIEW

File: prow/prowjobs/kubeflow/pipelines/kubeflow-pipelines-presubmits.yaml [hunk 1]
```diff
@@ -190,6 +190,16 @@ presubmits:
...
+  
+  - name: kubeflow-pipelines-manifests
+    cluster: build-kubeflow
+    decorate: true
+    run_if_changed: "^manifests/kustomize/.*$"
+    spec:
+      containers:
+      - image: gcr.io/google.com/cloudsdktool/cloud-sdk:latest
+        command:
+        - ./manifests/kustomize/hack/presubmit.sh
```
Comment (zijianjoy) [inline prow/prowjobs/kubeflow/pipelines/kubeflow-pipelines-presubmits.yaml:202]: I don't see the `presubmit.sh` file yet: https://github.com/kubeflow/pipelines/tree/master/manifests/kustomize/hack, are we going to add this file first?

Comment (zijianjoy) [reply prow/prowjobs/kubeflow/pipelines/kubeflow-pipelines-presubmits.yaml:202]: Ah I see the prerequisite PR: https://github.com/kubeflow/pipelines/pull/5390/files

Comment (Bobgy) [reply prow/prowjobs/kubeflow/pipelines/kubeflow-pipelines-presubmits.yaml:202]: Hi @zijianjoy, I was being a little lazy, there are no PRs other than the current one that is changing kustomize manifests. So I think it's fine to add the config change first, as long as the manifest PR looks good to you.

**Comment by Bobgy** [comment]
Thanks!
I'll unhold to experiment with it
/unhold

**Pull request merged**